resdk/tests/unit/test_resolwe.py

```diff
@@ -326,7 +331,7 @@ def test_call_with_all_args(self, resolwe_mock, os_mock, data_mock):
 class TestUploadFile(unittest.TestCase):
 
     def setUp(self):
-        self.fn = os.path.normpath(os.path.dirname(__file__) + "./../files/example.fastq")
+        self.file_path = os.path.normpath(os.path.dirname(__file__) + "./../files/example.fastq")
```

Comment:
You should ask the author. Here I have only removed warnings from the original implementation. I am now refactoring the original implementation to use a constant BASE_DIR where I use `abspath`:
```
BASE_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
```